charts/retool/templates/deployment_backend.yaml

```diff
@@ -77,21 +77,29 @@ spec:
             value: {{ template "retool.deploymentTemplateVersion" . }}
           - name: NODE_ENV
             value: production
-          {{- if include "retool.jobRunner.enabled" . }}
-          {{ if ( not $.Values.dbconnector.java.enabled ) }}
-          - name: DISABLE_JAVA_DBCONNECTOR
-            value: "true"
-          {{ end }}
+          {{ if and ( or (eq "latest" $.Values.image.tag ) ( semverCompare ">= 3.93.0-0" $.Values.image.tag ) ) ( .Values.jobRunner.enabled ) }}
```

Comment:
i think `semverCompare` might give you trouble with image tags that are like `3.xx.x-stable` or `-edge`

Reply:
I tested against the `-edge` ones and it works fine. I think it supports them because it's supposed to support SHAs there. That's why the comparison is against `3.93.0-0`.

I might add extra tests against a helm chart that has those versions just in case!